docs/src/main/paradox/scala/http/migration-guide/index.md

```diff
@@ -7,6 +7,6 @@
 
 * [migration-from-spray](migration-from-spray.md)
 * [migration-guide-2.4.x-10.0.x](migration-guide-2.4.x-10.0.x.md)
-* [migration-within-10.0.x-Api-May-Change](migration-within-10.0.x-Api-May-Change.md)
+* [migration-guide-10.0.x](migration-guide-10.0.x.md)
```

Comment:
My main worry is that at some point we will switch to Akka 2.5 and we'll need to document some hoops to jump through when moving from Akka 2.4, like migrating to use AbstractActor.